lib/hammer_cli_foreman/common_parameter.rb

```diff
@@ -26,6 +26,7 @@ class SetCommand < HammerCLIForeman::Command
 
       option "--name", "NAME", _("parameter name"), :required => true
       option "--value", "VALUE", _("parameter value"), :required => true
+      option "--hidden_value", "HIDDEN VALUE", _("should the value be hidden")
```

Comment:
Options should have dashes only:  `--hidden-value`. Multi-word param names should be connected with underscore: ` HIDDEN_VALUE`.